pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -686,9 +687,16 @@ func (c *Cluster) getPVCInitContainer(osdProps osdProperties) v1.Container {
 		Name:  blockPVCMapperInitContainer,
 		Image: c.spec.CephVersion.Image,
 		Command: []string{
-			"cp",
+			"/bin/sh", "-c",
+		},
+		Args: []string{
+			fmt.Sprintf(
+				"cp %s %s %s; test -e %s",
```

Comment:
It's intentional. If `test` is run only if `cp` succeeds`, this container fails due to EEXIST after restarting the OSD pod. I added `test -e <DEST>` as the alternative of EEXIST check because `cp` can't distinguish the type of errors and always returns 1 in failure. However, I found `-n` option, which avoids the overwrite in EEXIST case, is a better solution. I'll fix it.